src/igor/sync.go

```diff
@@ -82,14 +78,38 @@ func runSync(cmd *Command, args []string) {
 }
 
 func syncArista() {
+	// first get ground truth
+	gt, err := networkVlan()
+	if err != nil {
+		log.Fatal("Unable to acquire VLAN ground truth from arista")
+	}
 	// TODO: probably shouldn't iteration over .M directly
 	for _, r := range igor.Reservations.M {
 		if !r.IsActive(igor.Now) {
 			continue
 		}
 
+		w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
 		if !quiet {
-			fmt.Printf("set switchports for %v to %v\n", r.Hosts, r.Vlan)
+			// print all nodes, Igor VLANs, and arista VLANs
+			fmt.Fprintln(w, "NODE\tIGOR VLAN\tARISTA VLAN")
+			for _, host := range r.Hosts {
```

Comment:
You are asking to iterate over single element of an array within the loop of that array. This is not going to have the desired effect.